Makefile

```diff
@@ -153,3 +154,14 @@ usage:
 	@echo "    clean     - cleans the dist dir"
 	@echo "    distclean - cleans the dist dir and the dependencies"
 
+
+# Rules to attach checksums to JavaScript source in the header
```

Comment:
great job figuring out these rules and the sed pattern!

can you move this block above the help/usage section. this is just a style preference, that I like to have the usage at the bottom or the top not somewhere in-between. thanks